app/views/responsibility_mailer/remind_group.text.erb

```diff
@@ -1,17 +1,14 @@
-Sehr geehrte(r) Bearbeiter/-in von Meldungen in <%= Settings::Instance.name %>,
+<%= t :salutation_editor, name: Settings::Instance.name %>
 
-nachfolgende Meldung(en) wurde(n) in Ihre Zuständigkeit übergeben und ist/sind seit mindestens <%= @days %> Tag(en) noch nicht akzeptiert.
-
-Bitte prüfen Sie die Sachverhalte, akzeptieren Sie die Zuständigkeit und geben Sie den Bürgerinnen und Bürgern jeweils eine kurze Info zum Bearbeitungsstand mittels der öffentlichen Statusinformation – oder schließen Sie die Meldung(en) ab, falls dies bislang einfach vergessen wurde:
+<%= t('responsibility_mailer.remind_group.message') %>
```

Comment:
Mit dieser Änderung geht jetzt der Hinweis auf die Tage, die die Meldung bereits zugewiesen aber noch nicht akzeptiert wurde verloren. Das muss entweder zusammengefasst werden und um den `@days` Parameter ergänzt werden, oder werden zwei Einträge in der Übersetzung daraus. Definitiv müssen die Texte und Inhalte inkl. der Tagesangaben erhalten bleiben.